doc/state_estimation_nodes.rst

```diff
@@ -63,6 +63,10 @@ The default values for ``map_frame``, ``odom_frame``, and ``base_link_frame`` ar
 ^^^^^^^^^^^^^^^^^^^^^^
 Some packages require that your transforms are future-dated by a small time offset. The value of this parameter will be added to the timestamp of *map->odom* or *odom->base_link* transform being generated by the state estimation nodes in ``robot_localization``.
 
+~transform_timeout
```

Comment:
One last thing: can you add a note than non-zero values may affect the filter's timing? For example, if you tell it to output at 50 Hz, but it has to wait for a transformation for a given measurement, it will block, and you won't get 50 Hz.